src/utils.jl

```diff
@@ -1,3 +1,19 @@
+"""
```

Comment:
Sorry, just noticing this wasn't merged.

But one more quick comment before then. Can you rename this file sugar.jl or something?

We already confusingly have several other utils.jl files in subdirectories that make this a bit confusing.